Classes/Projection/ProjectorInterface.php

```diff
@@ -22,6 +22,16 @@
      * Returns the class name of the (main) Read Model of the concrete projector
      *
      * @return string
+     * @api
      */
     public function getReadModelClassName(): string;
+
+    /**
+     * Removes all objects of this repository as if remove() was called for all of them.
+     * For usage in the concrete projector.
+     *
+     * @return void
+     * @api
+     */
+    public function drop();
```

Comment:
Just wondering: Is "drop" a good name for this? Are we really dropping the projection? What about "reset"?